ddtrace/compat.py

```diff
@@ -22,6 +23,7 @@
 else:
     from queue import Queue
     from urllib.parse import urlencode
+    from urllib.parse import urlparse
```

Comment:
We have this down below already.

```python
from ...compat import parse

parse.urlparse(url)
```

Reply:
Right, hadn't noticed it before. Fixed now